### ISSUE TITLE
Suggestion: New badge

### ISSUE THREAD

**Issue by chalkygames123** [semantic-release/semantic-release#1825]
## New feature motivation

Thanks to #1715, the semantic-release's logo was added in [simple-icon v4.12.0](https://github.com/simple-icons/simple-icons/releases/tag/4.12.0) (https://github.com/simple-icons/simple-icons/pull/4995) and reflected in [shields.io](https://shields.io/), so we can now use a new badge.

## New feature description

_Not a feature_

## New feature implementation

Current:

[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)

```markdown
[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
```

Suggestion:

[![semantic-release](https://img.shields.io/badge/semantic--release-enabled-brightgreen?logo=semantic-release)](https://github.com/semantic-release/semantic-release)

```markdown
[![semantic-release](https://img.shields.io/badge/semantic--release-enabled-brightgreen?logo=semantic-release)](https://github.com/semantic-release/semantic-release)
```

BTW, does anyone know where the color `#e10079` used in the current one comes from?

**Comment by chalkygames123** [comment]
Sorry, I fixed a typo and added previews.

**Comment by chalkygames123** [comment]
While there are several possible variations, I prefer to place the brand name on the left and the status on the right, referring to a similar discussion, https://github.com/renovatebot/app-support/issues/38.

**Comment by travi** [comment]
again, apologies for missing this until now. 

> While there are several possible variations, I prefer to place the brand name on the left and the status on the right

i [went back and forth](https://github.com/semantic-release/semantic-release/pull/2235#issue-1046939669) on this as well. i was mainly trying to get just the icon on the left and the project name on the right, but was unsuccessful. i didnt really consider another word on the right, like "enabled", but like the approach. from a branding perspective, keeping the logo with the "signature" is usually a better approach anyway. i'm not sure that "enabled" feels like the right word to me, though. "enabled" feels like a simple on/off toggle to me, and semantic-release is a little bit more involved than that. is there another word that might be more appropriate, or am i over-thinking this?

> does anyone know where the color `#e10079` used in the current one comes from?

i [was curious about the same thing](https://github.com/semantic-release/semantic-release/pull/2235#issue-1046939669), but didn't end up following up further. i joined the project well after that was already defined, so i'm not familiar with the history. @gr2m do you know the background on this? do we want to continue to use the existing badge color or are we open to adjustment?

**Comment by gr2m** [comment]
I'm open to re-adjustment. 

Maybe @boennemann can shed some light on the history about the badges?

**Comment by chalkygames123** [comment]
@travi No problem at all, thank you for the feedback 😊

> "enabled" feels like a simple on/off toggle to me, and semantic-release is a little bit more involved than that. is there another word that might be more appropriate, or am i over-thinking this?

Yeah, I feel you. However, it is suitable as a word to include in the badge (which should be as short as possible) because of its generality, IMHO.

If you want to indicate the status of the release process specifically, you can add a separate "status badge" for each CI platform ([just like I do](https://github.com/chalkygames123/stylelint-config/tree/42edb9ccbe82c116e9dc1c262a95b1dc8237a4b1#readme)).

**Comment by boennemann** [comment]
> Maybe @boennemann can shed some light on the history about the badges?

Ohai!

Sooo, the badges.

I wanted a badge to indicate to contributors that they have to use conventional commits, and to maintainers that they should be aware of the consequences of merging a PR.

Iterating on this idea, it would probably be a good idea to mention the type of commit preset. This way it can provide even more actual value.

[![semantic-release](https://img.shields.io/badge/semantic--release-conventional--commits-e10079?logo=semantic-release)](https://github.com/semantic-release/semantic-release)
[![semantic-release](https://img.shields.io/badge/semantic--release-angular-e10079?logo=semantic-release)](https://github.com/semantic-release/semantic-release)

About the color. Back at the time there weren't many other badges around.
One of the most used ones was for whether dependencies are outdated.
So I was frequently scanning for whether the color changed from green to yellow or red.

With semantic-release the color is never changing and there is no status to indicate.
That's why I opted out of the red/yellow/red theme and went for the flashiest option.

**Comment by chalkygames123** [comment]
> Iterating on this idea, it would probably be a good idea to mention the type of commit preset. This way it can provide even more actual value.

Oh, that never crossed my mind, loving it!

> With semantic-release the color is never changing and there is no status to indicate.
That's why I opted out of the red/yellow/red theme and went for the flashiest option.

That's a fun fact and makes sense! So it might be better to keep rather than change the current color, which is unique and probably already widely used.

**Comment by travi** [comment]
> With semantic-release the color is never changing and there is no status to indicate.
> That's why I opted out of the red/yellow/red theme and went for the flashiest option.

thanks so much for the history details on this! makes complete sense and i think it is a good reason to keep the color

> Iterating on this idea, it would probably be a good idea to mention the type of commit preset. This way it can provide even more actual value.

fantastic suggestion! i think this could be helpful, not only to contributors to a project already configured to use semantic-release, but also to project maintainers looking to add semantic-release to their project since it is a reminder to step back and realize that semantic-release relies on a configured convention but does not define the rules of the convention.

i do have one concern about moving in that direction, though. we have plans to update the default convention from the angular convention to conventional-commits. while we will obviously communicate that type of change when we move forward with it, details like badges are often overlooked when accepting an upgrade like that. we should probably include a suggestion in the docs about the badge to try to remember to update the convention mentioned in the badge if the default changes in the future. 

that said, i could see value in working with [shields.io](https://shields.io/) to create a dynamic badge that could derive the convention from the project config (or use the default for that version of semantic-release if the project does not have config).

**Comment by travi** [comment]
> i could see value in working with [shields.io](https://shields.io/) to create a dynamic badge that could derive the convention from the project config (or use the default for that version of semantic-release if the project does not have config).

to be clear, i dont see this as a blocker to updating the suggestion for a static badge, just a potential nice follow up if someone is interested in pursuing it.

@chalkygames123 are you interested in opening a PR that updates the readme badge and the docs suggesting the badge to use in projects using semantic-release?

**Comment by chalkygames123** [comment]
> @chalkygames123 are you interested in opening a PR that updates the readme badge and the docs suggesting the badge to use in projects using semantic-release?

Yes, I'll do that later :)

**Comment by travi** [comment]
closed by #2266
